### DISCUSSION TITLE
Hidden and flashlight should not be incompatible in mania

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#19610] [Ideas]
currently mania doesnt have a memory based mod since both hidden and flashlight are reading mods (they just limit how much of the playfield you can see)
I understand *why* they arent (currently) compatible, you wouldnt be able to see anything, but that's the point. Similar to DDR's stealth mode where you can't see the notes, the whole point is pattern memorization. It would be a fun challenge and wouldnt do any harm, so why not?
Additionally, Taiko *does* allow HDFL even though it makes it impossible to see any hit objects, so there is already precedent for it in the game with a similar gamemode.

**Comment by NotGumballer91** [comment]
~~Combining them would add a free mod multiplier. I'd say it should be a separate fun mod itself.~~

Edit: I take that back. I just realized that every other rulesets allows Hidden and Flashlight mod combination, Except for mania.
This needs to be changed really.

**Reply by Doksha** [reply]
Also, just so you are aware, mania does not have mod multipliers in the first place. the only "multipliers" are mods that make the game easier (like HT and NF) which reduce score.
Neither HD nor FL have ever and likely will never affect the scoring system
